pkg/apis/openstackproviderconfig/v1alpha1/types.go

```diff
@@ -62,8 +67,24 @@ type RootVolume struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// OpenstackClusterProviderConfig is the providerConfig for OpenStack in the cluster object
+// +k8s:openapi-gen=true
+type OpenstackClusterProviderConfig struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// NodeCIDR is the OpenStack Subnet to be created
+	NodeCIDR string `json:"nodeCidr,omitempty"`
```

Comment:
Can you expand the docs here? It's not clear to me why we need the NodeCIDR here.

Reply:
Leaving this out as well at this point.

I will re-introduce it with a broader description when I add the reconciling code for networks. Does not make much sense to have fields that are actually not used.